axum-macros/src/from_request.rs

```diff
@@ -270,7 +288,7 @@ fn extract_each_field_rejection(
         }
     };
 
-    let impl_display = {
+    let impl_display = if rejection_derive_opt_outs.derive_display() {
```

Comment:
Could use `bool::then` (plus `.transpose()?`) here.

Reply:
Hm maybe. I think its fine as is.